diesel/src/sqlite/connection/raw.rs

```diff
@@ -230,19 +255,15 @@ extern "C" fn run_custom_function<F>(
                 internal_connection: conn,
             },
             None => {
-                ffi::sqlite3_result_error(
-                    ctx,
-                    NULL_DATA_ERR.as_ptr() as *const _ as *const _,
-                    NULL_DATA_ERR.len() as _,
-                );
+                context_error_str(ctx, NULL_DATA_ERR);
                 return;
             }
         };
         match f(&conn, args) {
```

Comment:
We should also have an `std::panic::catch_panic` here if we refactor this at all.